plugins/hbase/src/main/java/com/navercorp/pinpoint/plugin/hbase/interceptor/HbaseClientMethodInterceptor.java

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Copyright 2018 NAVER Corp.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.navercorp.pinpoint.plugin.hbase.interceptor;
+
+import com.navercorp.pinpoint.bootstrap.context.MethodDescriptor;
+import com.navercorp.pinpoint.bootstrap.context.SpanEventRecorder;
+import com.navercorp.pinpoint.bootstrap.context.TraceContext;
+import com.navercorp.pinpoint.bootstrap.interceptor.SpanEventSimpleAroundInterceptorForPlugin;
+import com.navercorp.pinpoint.plugin.hbase.HbasePluginConstants;
+
+import java.net.InetSocketAddress;
+
+/**
+ * The type Hbase client method interceptor.
+ *
+ * @author Victor.Zxy
+ * @version 1.8.1
+ * @since 2018/10/12
+ */
+public class HbaseClientMethodInterceptor extends SpanEventSimpleAroundInterceptorForPlugin {
+
+    /**
+     * Instantiates a new Hbase client method interceptor.
+     *
+     * @param traceContext the trace context
+     * @param descriptor   the descriptor
+     */
+    public HbaseClientMethodInterceptor(TraceContext traceContext, MethodDescriptor descriptor) {
+        super(traceContext, descriptor);
+    }
+
+    @Override
+    protected void doInBeforeTrace(SpanEventRecorder recorder, Object target, Object[] args) {
+        recorder.recordServiceType(HbasePluginConstants.HBASE);
+    }
+
+    @Override
+    protected void doInAfterTrace(SpanEventRecorder recorder, Object target, Object[] args, Object result, Throwable throwable) {
+        String endPoint = getEndPoint(args);
+        recorder.recordEndPoint(endPoint != null ? endPoint : "Unknown");
+        recorder.recordDestinationId(HbasePluginConstants.HBASE.getName());
```

Comment:
Just a note for the future, destination id should be a logical name of the destination node.
AFAIK, terminal nodes with the same destination id are merged into a single node in the server map, even though they have different end points. (Think of destination id == application name, end point == agent id for terminal nodes)

I'm not sure what would be a good value to use as destination id for hbase (maybe zk quorum + znode?), so we'll leave this as is for now.
If you have a good idea for this, please let me know.

Reply:
Thank U for your comments.
AFAIK, Hbase Client gets the real Region Server Address through the Zookeeper, then use RPC to operate(PUT/GET/SCAN/...) Region Server, so I think use the real address record end point. 

About the DestinationId, I refer to the implementation of Redis Plugin:
[recorder.recordDestinationId(RedisConstants.REDIS.getName());](https://github.com/naver/pinpoint/blob/master/plugins/redis/src/main/java/com/navercorp/pinpoint/plugin/redis/interceptor/JedisMethodInterceptor.java#L79)
What do you recommend as a DestinationId?

Reply:
Yes, you're absolutely correct, the client connects directly to region servers using the meta data stored in zookeeper. Your end point recording is correct.
I'm not sure what we should we use for destination id so we can go with `recorder.recordDestinationId(HbasePluginConstants.HBASE.getName());` for the purpose of this pull request.

If you do come up with a good value for the destination id, please let us know.

Reply:
OK